src/inspect_ai/tool/_tools/_computer/_computer.py

```diff
@@ -0,0 +1,200 @@
+import asyncio
+import shlex
+from typing import Awaitable, Callable
+
+from inspect_ai.model import Content
+from inspect_ai.tool import Tool, tool
+from inspect_ai.tool._tools._computer._computer_common import (
+    cursor_position,
+    double_click,
+    left_click,
+    left_click_drag,
+    middle_click,
+    mouse_move,
+    press_key,
+    right_click,
+    screenshot,
+    type,
+)
+
+# Export ToolResult from inspect_ai.tool
+ToolResult = str | int | float | bool | list[Content]
```

Comment:
Yes, we should indeed just export this from inspect_ai.tool

Reply:
I'll do that. Do you want that change split into a separate PR?

Reply:
NP with it being in this PR.

Reply:
It was exported, my VS Code couldn't find it to create the import for some reason. I want to review my imports in general. I think there are some cases I don't quite understand. For example, I find importing through `_` prefixed files a bit bogus. For example, I would expect to do:
```python
from inspect_ai.util import ContentText
```
instead of 
```python
from inspect_ai._util.content import ContentText
```

Reply:
Actually, the convention is that `_` prefixed modules are private. So if you have to resort to the `_` then there is something wrong. Are you using the Inspect VS Code extension? There is a tweak to the default settings we make there to tell VS Code to index 2 levels deep in Inspect which might not be in play (they hard code this for matplotlib and others, but we need to add ourselves manually to this list).